Add Ember + Storybook writeup (#894)
ember-learn/ember-blog: content/the-ember-times-issue-172.md

The [second video](https://www.youtube.com/watch?v=FOrCbXW223E) does an in-depth example of using TypeScript in Ember such as making a route, controller, and when using actions and decorators. We also briefly go through where to add global type declarations in your project and other tips for working with TypeScript in your Ember application today.

<insert youtube here? need to see if possible in substack>

---

## [Storybook for Ember tutorial 🎨](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/)

> [Storybook](https://storybook.js.org/) has taken the world of **design systems** by storm! What is Storybook? Storybook makes building UI components isolated from the business logic and context of your app easier, sort of a "style guide as a service". Check out the new [Storybook for Ember tutorial](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/) for guidance on setting up Storybook in your Ember app.

Reviewer: I thought the 2nd sentence was initially hard to understand (in particular, what "isolated" referred to). Maybe break it into two sentences?

```suggestion
[Storybook](https://storybook.js.org/) has taken the world of **design systems** by storm! What is Storybook? Storybook helps you build UI components that are isolated from business logic and context of your app. Sort of a "style guide as a service." Check out the new [Storybook for Ember tutorial](https://www.learnstorybook.com/intro-to-storybook/ember/en/get-started/) for guidance on setting up Storybook in your Ember app.
```

> [Thomas Gossmann (@gossi)](https://github.com/gossi) spearheaded the Ember + Storybook effort and wrote a complementary blog post, [Ember with Storybook – Behind the Scenes](https://gos.si/blog/ember-with-storybook-behind-the-scenes/). The post touches on how Storybook and Ember are independent build pipelines, options for writing Storybook docs in Markdown, and different strategies for Storybook setup depending on your app architecture (e.g. monorepo vs. multirepo). But wait, there's more - you can use @gossi's open source Ember design system, hokulea ([repo](https://github.com/hokulea/hokulea) and [Storybook](https://hokulea.netlify.app/), as a reference!

Reviewer: I think we can create the effect of a short paragraph by showing a list of items:

```suggestion
[Thomas Gossmann (@gossi)](https://github.com/gossi) spearheaded the Ember + Storybook effort and wrote a complementary blog post, [Ember with Storybook – Behind the Scenes](https://gos.si/blog/ember-with-storybook-behind-the-scenes/). The post touches on a few ideas:

- How Storybook and Ember are independent build pipelines
- Options for writing Storybook docs in Markdown
- Different strategies for Storybook setup depending on your app architecture (e.g. monorepo vs. multirepo).

But wait, there's more - you can use @gossi's open source Ember design system, hokulea, as a reference!

- [GitHub Repo](https://github.com/hokulea/hokulea)
- [Storybook](https://hokulea.netlify.app/)
```
